examples/checkout.c

```diff
@@ -0,0 +1,102 @@
+#include "common.h"
+#include <git2.h>
+#include <git2/checkout.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#ifndef _WIN32
+# include <pthread.h>
+# include <unistd.h>
+#endif
```

Comment:
Are these platform-specific headers even required? I can't see anything making use of them.